engine/src/desktop-menu.cpp

```diff
@@ -444,7 +444,8 @@ void MCButton::macopenmenu(void)
                 // SN-2014-08-25: [[ Bug 13240 ]] We need to keep the actual popup_menustring,
                 //  in case some menus are nested
                 MCStringRef t_menupick;
-                t_menupick = s_popup_menupick;
+                // PM-2015-02-09: [[ Bug 14521 ]] Make sure we don't pass nil to message_with_valueref_args
+                t_menupick = !MCStringIsEmpty(s_popup_menupick) ? s_popup_menupick : kMCEmptyString;
```

Comment:
Under the following scenario, MCPlatformPopUp returns true but s_popup_menupick is nil:
(Using develop-7.0 branch)
1. Create a new empty stack
2. Click "Inspector"
3. On the property inspector window that appears, click on the icon "Selection menu"
4. *Click* on "Inspect" or "Property Profile"